src/components/Countries/Countries.css

```diff
@@ -0,0 +1,6 @@
+.country-list {
+  display: grid;
+  min-height: 13rem;
+  width: 100%;
+  grid-template-columns: repeat(auto-fit, minmax(12rem, 1fr));
+}
```

Comment:
[OPTIONAL] I would recommend reducing the min-width size of the country div from `12rem` to `11rem` or `10rem` so your design matches the template design:

<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/160216866-aa00f53e-53b5-4f60-ba77-1752b8cb2686.png">

Reply:
fixed